.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+__tasks:__
+
+- [ ] test coverage
+
+__information:__
```

Comment:
I would change this to `##### Summary`. "information" doesn't really tell much - everything in the description of the PR is information. :)